CHANGELOG.md

```diff
@@ -25,8 +25,7 @@
   `utf_codepoint_to_int` functions.
 - Fixed `string.inspect`'s escaping of `"`, `\`, `\n`, `\r`, `\r\n`, and `\t`,
   which in turn fixes `io.debug`'s output of such strings.
-- The `bit_string` function in the `dynamic` module now knows how to handle
-  JavaScript `Uint8Array`s.
+- The `list` module gains the `group` function.
```

Comment:
It looks like one of the previous entries has been removed now, and the new entry is down as being in a version that has not yet been released. A rebase issue perhaps

Reply:
One should not rebase in a 5 min pause of a long meeting ;-)
Fixed it. I'm very sorry about that.